### ISSUE TITLE
windows are not black

### ISSUE THREAD

**Issue by YehorAltshuler** [BinaryInk/vscode-theme_dark-modern-oled#1]
windows are not black
![image](https://github.com/user-attachments/assets/bf54b92e-e2dc-4789-af60-3fc14c47c9d9)

**Comment by BinaryInk** [comment]
Hi, thanks for the feedback.

AFIAK, context windows are controlled by the OS; at least that's the case with Linux. Looks like that's MacOS if I'm not mistaken.

I'll double-check and look into it when I get a moment but I'm thinking there's nothing in VSCode's CSS to change that.

**Comment by BinaryInk** [comment]
I stand corrected, that particular context menu is _not_ controlled by the OS after all. It looked an _awful lot_ like a MacOS menu to me at first glance, but clearly it is not. I suppose I just never paid those particular menus much mind.

I'll see if I can't find what property controls that quick and make the change. Might be a little bit even after fixing before I get the extension updated in the VSCode Marketplace, but I'll post here what the property is so you can change it in your `settings.json` file temporarily in either case. 

If I can't find it in the next couple minutes, it might be a few days until I get around to digging into it; it's been a while since I've poked around in Code's CSS.

**Comment by YehorAltshuler** [comment]
Hi, this is mac os, but it's not an os window, i use cmd + . or ctrl + . On windows to open this window

**Comment by BinaryInk** [comment]
I found it, in addition to changing the code suggestions. I'm probably going to look for a couple other things I may have missed later today or this weekend before pushing out any changes since I haven't touched this in a while. 

You can add this to your settings.json in the meantime, or wait for the update, up to you.

```json
"workbench.colorCustomizations": {
		"editorActionList.background": "#000",
		"editorSuggestWidget.background": "#000",
},
```

Thanks again for the feedback!

**Comment by YehorAltshuler** [comment]
Thank you for your work

**Comment by BinaryInk** [comment]
Addressed this and some other backgrounds in v1.3.2
